op/model.py

```diff
@@ -91,6 +91,15 @@ def __init__(self, name, backend, cache):
     def __repr__(self):
         return f'<{type(self).__module__}.{type(self).__name__} {self.name}>'
 
+    @property
+    def is_leader(self):
+        if self.is_local:
```

Comment:
Let's please discuss the name of this attribute in today's call. Hopefully we can find something more appropriate with a quick brainstorm.

Reply:
Will introduce something like this in a separate PR per our discussion:

```
_is_our_unit
_is_our_app
```